src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -459,7 +459,7 @@ namespace winrt::TerminalApp::implementation
 
         _CreateNewTabFromSettings(profileGuid, settings);
 
-        const int tabCount = static_cast<int>(_tabs.size());
+        const int tabCount = static_cast<int>(_tabs.Size());
```

Comment:
Honestly, it looks like I should just be explicit here about the type. Since `Size()` here returns a `uint32_t`, I think I'll change the type to `const uint32_t`. Also, in this function, `tabCount` is only really used inside the `TraceLoggingInt32()` call right below, which I can just change to `TraceLoggingUInt32()`.